REINDEXING.md

```diff
@@ -0,0 +1,261 @@
+# Reindexing
+
+A reindex operation runs the source data from the [adapters](docs/adapters/README.md) through the pipeline causing it to be re-transformed / matched & merged as appropriate.
+
+## How to run a reindex
+
+Running a reindex will require:
+
+- Preparing and deploying a release
+- Terraforming a new pipeline using that release
+- Running the reindex script to feed the new pipeline
+
+### Update the catalogue-pipeline environments
+
+The catalogue-pipeline is deployed using [weco-deploy](https://github.com/wellcomecollection/weco-deploy) but does not deploy in the usual way as there is often one release per pipeline. 
+
+Each environment will represent a new release.
+
+In `.wellcome_project`:
+
+```
+catalogue_pipeline:
+  environments:
+    - id: "2021-06-27"
+      name: "2021-06-27"
+    - id: "2021-07-06"
+      name: "2021-07-06"
+    - id: "MY-NEW-PIPELINE"
+      name: "MY-NEW-PIPELINE"
+```
+
+Name your new pipeline environment using the date pattern indicated above. You will need to use the environment label when terraforming a new pipeline.
+
+### Prepare a release
+
+Prepare a release following the example below. You may want to choose a specific git ref as a label and _not_ latest.
+
+```
+> weco-deploy --project-id catalogue_pipeline prepare --from-label latest --description "A helpful description"
+
+Prepared release from images in latest
+Requested by: arn:aws:sts::760097843905:assumed-role/platform-superdev/KennyR@Wellcomecloud.onmicrosoft.com
+Date created: 2021-07-08T15:11:48.403652
+
+service                old image    new image    Git commit
+---------------------  -----------  -----------  ------------
+aspect_ratio_inferrer  efd451f      -            -
+batcher                532ddee      -            -
+feature_inferrer       efd451f      -            -
+id_minter              021df38      -            -
+inference_manager      021df38      -            -
+ingestor_images        021df38      -            -
+ingestor_works         021df38      -            -
+matcher                021df38      -            -
+merger                 021df38      -            -
+palette_inferrer       efd451f      -            -
+relation_embedder      021df38      -            -
+router                 021df38      -            -
+transformer_calm       021df38      -            -
+transformer_mets       021df38      -            -
+transformer_miro       021df38      -            -
+transformer_sierra     021df38      -            -
+transformer_tei        f9f910d      -            -
+
+Created release eeb9482f-0a6a-4211-8de9-27cee7567134
+```
```

Comment:
I would trim the output just to show the important parts, and the command – the exact output isn't particularly relevant to running a reindex, and will get out-of-date quickly.